### DISCUSSION TITLE
FieldTypeFunc' is not assignable to type

### DISCUSSION THREAD

**Discussion by devmnj** [keystonejs/keystone#6610] [Event Q&A Q3 2021]
I go the error 
**FieldTypeFunc' is not assignable to type**
`';
import { text } from '@keystone-next/fields';
import { document } from  '@keystone-next/fields-document'
// const Post = list({
//   fields: {
    
//     title: text({ isRequired: true }),
//     slug: text(),
//     content: text(),
     
//   },
// });
export const lists = createSchema({
  Post: list({
    fields: {
      content: document({
        formatting: true,
        dividers: true,
        links: true,
        layouts: [
          [1, 1],
          [1, 1, 1],
        ],
      }),
    },
  }),
});`

can anybody help me?

**Comment by raveling** [comment]
Hey @manojap,

Thanks for your question! It looks like you're stuck making progress with your project.

To get the most out of the live event, we're keeping this Community Q&A category focused on questions that are relevant to the community as a whole.

Seeing as this question is an error specific to your project, would you mind posting it to our community Slack in the #keystone-6 channel?

That's the best place to get focused support on blocks you encounter with Keystone - https://community.keystonejs.com
